kloppy/domain/models/event.py

```diff
@@ -804,6 +812,7 @@ def add_state(self, *builder_keys):
 
     def to_pandas(
```

Comment:
The signature of the `to_pandas` function is becoming increasingly complicated, especially for users who have limited programming experience. The arguments `*fields`, `record_converter` and `additional_columns` all accept a `Callable` that operates on an instance of the `Event` class.

Reply:
Oh nooesh.. I forgot to push latest commit. That commit reverts the changes to `to_pandas`.